defaults/main.yml

```diff
@@ -69,7 +69,7 @@ rhel_07_010230: true
 rhel_07_010240: true
 rhel_07_010250: true
 # Potential for locking out users
-rhel_07_010260: false
+rhel_07_010260: true
```

Comment:
I'm on the fence about enabling this by default. It changes the known behavior of this role and still has the potential to lock people out of their AWS instances...just not immediately.

Reply:
Okay, I'm fine merging all but the last commit.  That's why I made it a separate commit.  The role won't lock the root account.  See the awk changes earlier.

Reply:
Ok. If you remove that last commit from this PR I'll merge it in.

Reply:
ok, done.